apps/prairielearn/src/pages/instructorAssessmentDownloads/instructorAssessmentDownloads.ts

```diff
@@ -69,14 +235,21 @@ async function pipeCursorToArchive(res, cursor: sqldb.CursorIterator<any>) {
 
   for await (const rows of cursor.iterate(100)) {
     for (const row of rows) {
-      let contents: string | Buffer;
-      try {
-        contents = Buffer.from(typeof row.contents === 'string' ? row.contents : '', 'base64');
-      } catch {
-        // Ignore any errors in reading the contents and treat as a blank file.
-        contents = '';
+      // Sort files to ensure consistent ordering; this is done
+      // for backwards compatibility and may not be necessary.
```

Comment:
👀 `files_for_manual_grading` had `filename` in an `ORDER BY` clause. I don't know if this is actually needed; it's probably safe to remove?

Reply:
I agree that it's probably not strictly necessary, but I'm also a fan of always returning data with a consistent ordering and this seems cheap and easy.